include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -207,6 +207,33 @@ class __scan_single_wg_kernel;
 template <typename... _Name>
 class __scan_single_wg_dynamic_kernel;
 
+namespace __scan_detail
```

Comment:
I don't think we need extra namespace (unless we have other precedence in the library). You can just put it closer to scan submitters.

Reply:
Fixed.